packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -724,30 +720,168 @@ class TimeInput extends React.Component {
       { 'initial-focus': this.state.secondInitialFocused },
     ]);
 
-    const formatDescriptionId = `terra-time-input-description-format-${this.uuid}`;
-
-    const format = showSeconds
-      ? `(${intl.formatMessage({ id: 'Terra.timeInput.hh' })}:${intl.formatMessage({ id: 'Terra.timeInput.mm' })}:${intl.formatMessage({ id: 'Terra.timeInput.ss' })})`
-      : `(${intl.formatMessage({ id: 'Terra.timeInput.hh' })}:${intl.formatMessage({ id: 'Terra.timeInput.mm' })})`;
+    function mask() {
+      if (showSeconds) {
+        return intl.formatMessage({
+          id: 'Terra.timeInput.maskHourMinute',
+          defaultMessage: '(hh:mm:ss)',
+          description: 'A visual hint that the Time Input expects two-digit hour, minute and second.',
+        });
+      }
+      return intl.formatMessage({
+        id: 'Terra.timeInput.maskHourMinuteSecond',
+        defaultMessage: '(hh:mm)',
+        description: 'Like Terra.timeInput.maskHourMinute but no second.',
```

Comment:
Should we update the description to be similar to the one above like this instead of referencing Terra.timeInput.maskHourMinute?

`A visual hint that the Time Input expects two-digit hour and minute.`

Reply:
Will do. I must have been tired when I coded this 😄 .